src/main.ts

```diff
@@ -0,0 +1,23 @@
+import { createHelpScoutClient } from './helpScoutClient';
+import { createService } from './createService';
+
+const githubWebhookSecret = process.env.GITHUB_WEBHOOK_SECRET || 'mySuperSecretToken';
+
+const helpScoutAppId = process.env.HELP_SCOUT_APP_ID || '';
+const helpScoutAppSecret = process.env.HELP_SCOUT_APP_SECRET || '';
+
+const helpScoutMailboxes = (process.env.HELP_SCOUT_MAILBOXES &&
+  JSON.parse(process.env.HELP_SCOUT_MAILBOXES)) || {
```

Comment:
json parse will error here, so fallback is never called

Reply:
Fallback is used when the value is not provided, if provided then we parse.